app/templates/admin/userManagement.html

```diff
@@ -134,7 +134,7 @@ <h3 class="accordion-header" id="headingThree">
                 <tr class="p-1">
                     <td> <input type="text" name="Name" value ="{{program.programName}}" required> </td>
                     <td><input type="text" name="Email" value="{{program.emailReplyTo}}" required></td>
-                    <td><input type="text" name="Sender" value="{{program.emailSenderName}}" required></td>
+                    <td><input type="text" name="Sender" value="" required></td>
```

Comment:
Now the input box is always empty after the name is updated. We want the box to be empty when they haven't inserted a name yet but if they have one in there it should show them as the sender.

Reply:
It also shouldn't be required. They might just want to update the program name